File: 169.cc
#include <algorithm>
#include <iostream>
#include <vector>
using std::cout;
using std::endl;
using std::sort;
using std::vector;

class Solution {
public:
    int majorityElement(vector<int>& nums) {
        sort(nums.begin(), nums.end());
        return nums[nums.size() / 2];
    }
};

int main() {
    vector<int> nums = {1, 2, 2};
    cout << Solution().majorityElement(nums) << endl;
    return 0;
}

File: 169.py
class Solution:
    def majorityElement(self, nums):
        """
        :type nums: List[int]
        :rtype: int
        """
        return sorted(nums)[len(nums) // 2]


if __name__ == "__main__":
    print(Solution().majorityElement([1, 2, 2]))
